cmake.py

```diff
@@ -18,11 +19,18 @@ def fetch_subprojects(state):
     if state['subprojects']:
         contents += textwrap.dedent(
         """
+        if ( NOT GIT_EXECUTABLE )
+            find_package( Git 2.1 )
+            if ( NOT GIT_FOUND )
+                message( FATAL_ERROR "git installation was not found." )
+            endif()
+        endif()
+
         if( NOT ROOT_DIRECTORY )
             set( ROOT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} )
             if ( NOT fetched_subprojects )
                 if ( NOT PYTHON_EXECUTABLE )
-                    find_package( PythonInterp )
+                    find_package( PythonInterp 3.4 )
```

Comment:
I wonder how strict we want to be on the Python version right now. While we do stay that we require Python 3.4, there is nothing we do that requires Python 3.x. @apmccartney @whaeck care to weigh in on this?